### ISSUE TITLE
field color oddity

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/SignStream3#471]
Don't know if this is related to #452, but not clear why "adv" shows up in red here

<img width="979" alt="Screen Shot 2021-05-29 at 1 23 20 PM" src="https://user-images.githubusercontent.com/13629122/120079310-407a4080-c081-11eb-9ac1-d002177ca212.png">

<img width="584" alt="Screen Shot 2021-05-29 at 1 25 23 PM" src="https://user-images.githubusercontent.com/13629122/120079346-5f78d280-c081-11eb-932f-b18a2afb190f.png">

And different field names turn successively red and stay red for a while...

<img width="1003" alt="Screen Shot 2021-05-29 at 1 26 28 PM" src="https://user-images.githubusercontent.com/13629122/120079408-a1097d80-c081-11eb-882b-0de332e8fa40.png">

<img width="736" alt="Screen Shot 2021-05-29 at 1 27 02 PM" src="https://user-images.githubusercontent.com/13629122/120079413-a49d0480-c081-11eb-932e-7fd8184b1dfd.png">

**Comment by douglas-motto-at-rutgers** [comment]
This behavior is the same or at least similar to 3.3.1 and 3.3.5.

Seems to appear when right clicking on event.

Marking as enhancement.

**Comment by cneidle** [comment]
No.  I am not right-clicking on anything.  And this -- the field name on the left appearing in red when it is not selected -- does not happen in 3.3.5.

**Comment by douglas-motto-at-rutgers** [comment]
Tell me the exact steps you do.  

As I see it..

-right clicking on event.
-double clicking on a field line...in an empty spot.

In 3.3.5 and the newest versions highlights the field label in red.

**Comment by cneidle** [comment]
What turned the left field label red for me was changing the value (by double-clicking on the displayed value), and then it seems to stick for a while until something else turns red.  That obviously should not happen.

You discovered other things that trigger this behavior, although they also obviously should not.

**Comment by douglas-motto-at-rutgers** [comment]
OK...double clicking to change the value and the label turning red...is also observed similarly in 3.3.5 and in the new versions.

I'm leaving this issue open as something new to change.

To NOTE: I also noticed this unusual red highlight behavior a few months ago and it is listed internally as a lower priority item to discuss.  I also included it in broader discussion about highlighting and selecting fields, labels, and events in general.

**Comment by cneidle** [comment]
Adding label "BUG - but also in previous versions" to distinguish from requests for new features or functionalities.

**Comment by cneidle** [comment]
This is  not an enhancement. It  is a bug that has existed for a while. It is a bug that is not critical to fix before the next release.
